### PR TITLE
add java8 branch & add LD_LIBRARY_PATH env variable

### PR DESCRIPTION
**Pull request by vangie**
1. 新建 java8 branch 用户构建 java8 docker image
2. 增加 aliyun 的 sources.list 以加速安装 apt package
3. 添加 LD_LIBRARY_PATH 环境变量

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,54 +1,49 @@
...
-        --registry http://registry.npm.taobao.org \
-        co@4.6.0 \
-        gm@1.23.0 \
-        ali-oss@4.10.1 \
-        aliyun-sdk@1.10.12 \
-        @alicloud/fc@1.2.2 \
-        opencv@6.0.0 \
-        tablestore@4.0.4
-
-RUN npm cache clean --force
```
Comment (rsonghuster) [inline Dockerfile:43]: 这个不是java的吗？之前为什么是node的

Comment (vangie) [reply Dockerfile:43]: 因为这个 branch 是新建的，之前的代码来自于 master 。master 上的 Dockerfile 是 nodejs 版本的。

File: Dockerfile [hunk 2]
```diff
@@ -1,54 +1,49 @@
...
-
-# Remove package.json
-RUN rm -f package.json
+# Copy all files to home directory.
+COPY pom.xml .
```
Comment (nerdyyatrice) [inline Dockerfile]: as we discussed, looks like we don't need this

File: Dockerfile [hunk 3]
```diff
@@ -1,54 +1,49 @@
...
 
 # Generate usernames
 RUN for i in $(seq 10000 10999); do \
         echo "user$i:x:$i:$i::/tmp:/usr/sbin/nologin" >> /etc/passwd; \
     done
 
-# Start a shell by default
-CMD ["bash"]
+#download all the dependencies
+RUN ["mvn", "clean", "install"]
```
Comment (nerdyyatrice) [inline Dockerfile]: as we discussed, looks like we don't need these two either

File: Dockerfile [hunk 4]
```diff
@@ -1,54 +1,49 @@
...
 
-# Install server dependencies.
-RUN npm install \
-        --loglevel error \
-        --registry http://registry.npm.taobao.org
+RUN mv /etc/apt/sources.list /etc/apt/sources.list.bak
+COPY sources.list /etc/apt/
 
 # Install common libraries
 RUN apt-get update && apt-get install -y \
```
Comment (nerdyyatrice) [inline Dockerfile:29]: those install usually takes a long time within GFW, I guess this is one time thing?

**Comment by nerdyyatrice** [comment]
LGTM

**Pull request merged**